test/it/helpers/browser.js

```diff
@@ -1,102 +1,191 @@
-const WAIT_UNTIL_TIMEOUT_MILLIS = 10000
-const WAIT_UNTIL_INTERVAL = 300
+import { assert } from 'chai'
 
-export function sendEvent (event, expectedRequests, server) {
-  const json = JSON.stringify(event)
-  try {
-    browser.execute(`window.liQ = window.liQ || [];window.liQ.push(${json})`)
-    waitForRequests(expectedRequests, server)
-  } catch (e) {
-    console.error(e, server.getHistory().length, expectedRequests)
-  }
+const WAIT_UNTIL_TIMEOUT_MILLIS = 30000
+const WAIT_UNTIL_INTERVAL = 500
+
+export async function getText (selector) {
+  var text
+  await browser.waitUntil(
+    async () => {
+      try {
+        const element = await $(selector)
+        if (element.elementId) {
```

Comment:
Seems that after dom manipulation it might return an element with an undefined elementid. Trying to call getText on that will blow up